src/plugins/ceramic-feed/server/content-types/ceramic-post/lifecycles.js

```diff
@@ -154,7 +156,26 @@ const LifecycleHooks = new class {
     const { relatedFieldName } = metadata
     const entitySchema = schemaName ? schemas[schemaName] : schema
     const { fields, mediaFields, relationFields } = entitySchema
-    const payload = pick(entity, fields)
+
+    const payload = pick(
+      entity,
+      array.remove(fields, 'publishWallet', 'publishDapp') //converting to tags
+    )
+
+    //handle tags for orbis
+    if (fields.includes('publishWallet')) {
```

Comment:
with an tags array constant this could be replaced via forEach over the array

Reply:
done